dataactcore/models/jobTrackerInterface.py

```diff
@@ -241,16 +240,20 @@ def getNumberOfRowsById(self,jobId):
         """ Get number of rows in file for job matching ID """
         return self.getJobById(jobId).number_of_rows
 
+    def getNumberOfValidRowsById(self, jobId):
+        """Get number of file's rows that passed validations."""
+        return self.getJobById(jobId).number_of_rows_valid
```

Comment:
It does not affect list_submissions, as that's going by total number of errors in error DB, not number of valid rows.